magellan-library/src/main/java/com/wealthfront/magellan/core/Navigable.kt

```diff
@@ -0,0 +1,6 @@
+package com.wealthfront.magellan.core
+
+import com.wealthfront.magellan.lifecycle.LifecycleAware
+import com.wealthfront.magellan.view.Displayable
+
+interface Navigable : LifecycleAware, Displayable
```

Comment:
Introducing `Navigable` which our screens will implement, this allows us to have multiple implementations of screens. For right now, we can have a screen which depends on view binding and another in the future for jetpack compose.